src/modules/i18n/components/__tests__/I18nProvider.spec.js

```diff
@@ -42,29 +42,23 @@ describe('I18nProvider', () => {
   })
 
   it('should set content language if not yet set', async () => {
-    const mockGetLocale = () => 'de_DE'
     const appSettings = new AppSettings()
     const mockSetContentLanguage = jest.fn()
 
-    render(<I18nProvider setContentLanguage={mockSetContentLanguage}
-                         getLocale={mockGetLocale}
-                         appSettings={appSettings} />)
+    render(<I18nProvider setContentLanguage={mockSetContentLanguage} />)
     await waitForExpect(async () => {
       expect(mockSetContentLanguage).toHaveBeenCalledTimes(1)
-      expect(mockSetContentLanguage).toHaveBeenCalledWith('de')
-      expect(await appSettings.loadContentLanguage()).toBe('de')
+      expect(mockSetContentLanguage).toHaveBeenCalledWith('en')
+      expect(await appSettings.loadContentLanguage()).toBe('en')
     })
   })
 
   it('should not set content language if already set', () => {
```

Comment:
You have to use the `async () => {` syntax here and `await waitForExpect(async () => {`! Otherwise the test will always pass, even if the expects inside of `waitForExpect` don't!

Reply:
True :P https://github.com/Integreat/integreat-react-native-app/commit/74566bdc002f8668b4aaa5521b13ed2a9bf1f990